Foundation/Collection/Collection.hs

```diff
@@ -157,3 +159,11 @@ instance Collection c => Collection (NonEmpty c) where
     minimum = minimum . getNonEmpty
     all p = all p . getNonEmpty
     any p = any p . getNonEmpty
+
+-- | Return True if all the elements in the collection are True
+and :: (Collection col, Element col ~ Bool) => col -> Bool
+and = all (== True)
+
+-- | Return True if at least one element in the collection is True
+or :: (Collection col, Element col ~ Bool) => col -> Bool
+or = any (== True)
```

Comment:
You might want to override the hlint error here below the [line 82 of of .hlint.yaml](https://github.com/haskell-foundation/foundation/blob/master/.hlint.yaml#L82)